src/main/java/com/worksap/fig/lang/Hash.java

```diff
@@ -117,4 +117,47 @@ default boolean containsAny(BiPredicate<K, V> condition) {
         }
         return this;
     }
+
+    /**
+     * Returns the keys of the given value.
+     * @param value the value
+     * @return the collection of keys whose value is the given value
+     */
+    default Seq<K> getKey(V value) {
```

Comment:
In Java, getXXX always means the getter of an field, this has become a convention. So we'd better use other names, for example: `keysOf`